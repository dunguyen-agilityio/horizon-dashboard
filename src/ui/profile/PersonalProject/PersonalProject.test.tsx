import { render } from '@testing-library/react';

import PersonalProjects from '.';

describe('PersonalProjects tests', () => {
  it('Should match snapshot', () => {
    const { container } = render(<PersonalProjects userId={3} />);
    expect(container).toMatchSnapshot();
  });

  it('Should render correct title and description', () => {
    const { getByText } = render(<PersonalProjects userId={3} />);

    const title = getByText('All Projects 3');
    const description = getByText(
      'Here you can find more details about your projects. Keep you user engaged by providing meaningful information.',
    );
    expect(title).toBeInTheDocument();
    expect(description).toBeInTheDocument();
  });
});
