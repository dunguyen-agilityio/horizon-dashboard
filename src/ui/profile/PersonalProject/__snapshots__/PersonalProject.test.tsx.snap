// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalProjects tests Should match snapshot 1`] = `
<div>
  <div
    class="dark:bg-indigo bg-white w-[350px] sm:w-full lg:w-[552px] py-6 px-8 rounded-md"
  >
    <div
      class="text-center xl:text-start"
    >
      <p
        class="text-primary dark:text-white text-1.5xl leading-xl font-extrabold leading-8 mb-4"
      >
        All Projects 
        3
      </p>
      <p
        class="text-secondary dark:text-secondary text-medium leading-xl leading-[26px] tracking-tight mb-8"
      >
        Here you can find more details about your projects. Keep you user engaged by providing meaningful information.
      </p>
    </div>
    <div
      class="flex flex-col gap-5 mb-2"
    />
  </div>
</div>
`;
